README.md

```diff
@@ -891,16 +891,32 @@ INFO[2190] Got new data stream
 DEBU[2190] Content update                                update=content update: 823f79f2-0001-4eb2-9ba0-2a8c1b284443 - 93a17ce2-788d-476f-bd11-a5580a2f35f3
 content: "content"
 commands:
-- Delete ("domain-addresses"/"good"/"example.com")
-- Add ("domain-addresses"/"good"/"example.com")
-- Delete ("domain-addresses"/"bad"/"example.com")
-- Add ("domain-addresses"/"bad"/"example.com")
+- Delete Path ("domain-addresses"/"good"/"example.com")
+- Add Path ("domain-addresses"/"good"/"example.com")
+- Delete Path ("domain-addresses"/"bad"/"example.com")
+- Add Path ("domain-addresses"/"bad"/"example.com")
 INFO[2190] Got apply command
 INFO[2190] Content update has been applied               cid=content curr-tag=93a17ce2-788d-476f-bd11-a5580a2f35f3 id=5 
 ...
 ```
 
-Contents with different ids and policies can be updated independently and in paralel.
+Contents with different ids and policies can be updated independently and in parallel.
+
+## Debugging Options
```

Comment:
I think that as policies has tree-like structure it's better to reflect the fact in API.

So `GET /storage/rootPID/innerPID/moreInnerPID/...` gives exact policy or rule by path (and you don't need to provide separate endpoints for policies, policy sets and rules). You can control how many nested policies and rules you get with query parameter. for example `GET /storage/rootPID?depth=2` gives root policy itself and all its immediate children.

For search I suggest to have separate endpoint `/search` with query parameter id. It should use "contains" comparison to get matching policies. And return list of full paths to them. So `GET /search?id=xxx` returns "/yxxxz", "/yxxxz/xxx" but not "/yxxxz/yyy". In unix shell command `find / -name "*xxx*"` works very similar to that. Here you also can add parameters such as depth to limit nesting search goes down and limit parameter to limit number of results.

You need to take special measures to not eat all memory during rendering policies as a response.